src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventSearchControllerTest.java

```diff
@@ -103,6 +103,7 @@ void returnsAllFieldsCorrectly() throws Exception {
         var response = json.from(mvcResult.getResponse().getContentAsString());
         assertThat(response).hasJsonPathNumberValue("[0].id");
         assertThat(response).hasJsonPathStringValue("[0].created_at");
```

Comment:
You've removed created_at from this endpoint so think should be removed from this test.